app/src/main/java/com/example/playlistmaker/data/dto/GetClickedTrackFromGson.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.playlistmaker.data.dto
+
+import com.example.playlistmaker.domain.api.GetClickedTrackFromData
+import com.example.playlistmaker.domain.models.ClickedTrack
+import com.google.gson.Gson
+
+class GetClickedTrackFromGson(): GetClickedTrackFromData {
```

Comment:
```suggestion
class GetClickedTrackFromGsonUseCase(): GetClickedTrackFromDataUseCase {
```
UseCase'ы принято называть таким образом, чтобы в конце шла строка "UseCase", например:
`GetDataUseCase`
`SaveDataUseCase`